Firestore/Source/Local/FSTLevelDBMutationQueue.mm

```diff
@@ -460,9 +459,9 @@ - (nullable FSTMutationBatch *)nextMutationBatchAfterBatchID:(FSTBatchID)batchID
       if (mutationIterator->Valid()) {
         foundKeyDescription = [FSTLevelDBKey descriptionForKey:mutationIterator->key()];
       }
-      FSTFail(
-          @"Dangling document-mutation reference found: "
-          @"Missing batch %@; seeking there found %@",
+      HARD_FAIL(
+          "Dangling document-mutation reference found: "
+          "Missing batch %@; seeking there found %s",
```

Comment:
The first specifier should be `%s`, not `%@`.

Reply:
Done.